app/assets/stylesheets/views/_homepage.scss

```diff
@@ -159,3 +159,48 @@
   @include govuk-font(36, $weight: bold);
   color: govuk-colour("white");
 }
+
+.homepage-links-and-search {
+  background: govuk-colour("light-grey");
+  padding-bottom: govuk-spacing(8);
+  padding-top: govuk-spacing(6);
+
+  @include govuk-media-query($from: "desktop") {
+    padding-bottom: govuk-spacing(8);
+  }
+}
+
+.homepage-search {
+  @include govuk-media-query($until: "tablet") {
+    max-width: 420px;
+  }
+}
+
+.homepage-most-viewed-list {
```

Comment:
Whilst it doesn't add a huge amount here, using `govuk-list` alongside this class to dictate spacing feels apt as the element is a list in this case, not something we want to just use list markup for. By my calculation it would mean we're writing less CSS here and using a class that get's loaded anyway.

Reply:
Using `govuk-list` and then overriding the spacing was the reason for the [recent layout bug in the navigation header][1] - each page having two  stylesheets means that the order of the selectors can't always be controlled.

With that in mind, I'd rather avoid using `govuk-list` when anything needs to be overridden.

[1]: https://github.com/alphagov/govuk_publishing_components/pull/2460

Reply:
Gotcha, I'll back down on this one. Resolving.